webapp/portlet/webpack.dev.js

```diff
@@ -6,7 +6,7 @@ const webpackCommonConfig = require('./webpack.common.js');
 const app = 'social-portlet';
 
 // add the server path to your server location path
-const exoServerPath = "/exo-server";
+const exoServerPath = "/home/exo/work/exoRepos/plf-enterprise-tomcat-standalone-6.2.x-20210127.094402-162/platform-6.2.x-SNAPSHOT";
```

Comment:
restore the default configuration please.

Reply:
restored!